src/lang/english.txt

```diff
@@ -2859,6 +2859,11 @@ STR_HOUSE_PICKER_CLASS_ZONE3                                    :Outer Suburbs
 STR_HOUSE_PICKER_CLASS_ZONE4                                    :Inner Suburbs
 STR_HOUSE_PICKER_CLASS_ZONE5                                    :Town centre
 
+STR_HOUSE_PICKER_PROTECT_TITLE                                  :Protect
```

Comment:
```suggestion
STR_HOUSE_PICKER_PROTECT_TITLE                                  :Avoid demolishing
```

Maybe? The more I think about it, the less clear `Protect` is from a user perspective. Especially if it has very weird rules like "only for towns and AIs".

But dunno if this suggestion is better :D